crates/cli/src/config_files/build.rs

```diff
@@ -46,7 +47,7 @@ impl Build {
             let mut wasm_buf = Vec::new();
             File::open(&artifact_path)?.read_to_end(&mut wasm_buf)?;
 
-            Ok(base64::encode(&wasm_buf))
+            Ok(DnaWasm::from_bytes(wasm_buf))
```

Comment:
We were actually not even using DnaWasm's Serialize methods to output WASM; we were just serializing to base-64, and it happened to be compatible...